sources/packages/backend/libs/integrations/src/esdc-integration/e-cert-integration/models/e-cert-integration-model.ts

```diff
@@ -45,6 +45,7 @@ export interface ECertRecord {
   awards: Award[];
   stopFullTimeBCFunding: boolean;
   courseLoad?: number;
+  tuitionRemittanceEffectiveAmount: number;
```

Comment:
No changes needed if we follow https://github.com/bcgov/SIMS/pull/1840/files#r1152324927